### PR TITLE
Update CrowdStrike Risk Names

### PR DESCRIPTION
**Pull request by UNC1739**

### CODE REVIEW

File: pkg/asm/capabilities/crowdstrike.go [hunk 1]
```diff
@@ -112,36 +112,61 @@ func (task *Crowdstrike) Invoke() error {
...
 	}
 
-	rules := []struct {
-		Platform string
-		Setting  string
-		RiskName string
-	}{
-		{"Windows", "Ransomware", "windows-ransomware-control-disabled"},
-		{"Linux", "Enhanced Visibility", "linux-prevention-control-disabled"},
+	rules := make(map[string]map[string]map[string]string)
```
Comment (privateducky) [inline pkg/asm/capabilities/crowdstrike.go:115]: is this PR solely to put plain-text names on crowdstrike risks, or is something else hiding in here?

Comment (UNC1739) [reply pkg/asm/capabilities/crowdstrike.go:115]: Yeah it's just to update the risk names so they are more specific and it's easier to understand from the risk name what the issue that is being reported it.

Comment (privateducky) [reply pkg/asm/capabilities/crowdstrike.go:115]: yep; in that case, let's defer this topic until: https://github.com/praetorian-inc/chariot-ui/issues/658

in that ticket, we'll be allowing you to generically rename risks.

**Comment by UNC1739** [comment]
@privateducky I think this is different than the ticket you linked. The issue is now that we are using the asset DNS name for reporting risks we need to use distinct identifiers for the risk names. If two ransomware controls are disabled we will report both of them as windows-ransomware-control-disabled and thus one of them won't be reported properly because it will be deduped out.

**Comment by UNC1739** [comment]
Sorry that might have been the "is there something hiding there portion" :)

**Comment by privateducky** [comment]
> Sorry that might have been the "is there something hiding there portion" :)

LOL yea, that's why I asked haha

**Comment by UNC1739** [comment]
> > Sorry that might have been the "is there something hiding there portion" :)
> 
> LOL yea, that's why I asked haha

I'm fine with moving back to an array and removing the tolowercase stuff if you aren't a fan of that. The lowercase was more just general bulletproofing and the map was just to not have to do so many loops to parse all the config options and map them to risks.

**Pull request merged**